instance/models.py

```diff
@@ -147,6 +156,7 @@ class WriteitInstancePopitInstanceRecord(models.Model):
         )
     writeitinstance = models.ForeignKey(WriteItInstance)
     popitapiinstance = models.ForeignKey(ApiInstance)
+    popolo_source = models.ForeignKey(PopoloSource, null=True, blank=True)
```

Comment:
In fact it's just missing the import - we need this foreign key to be there for the datamigration which will populate it with the equivalent PopoloSource to the old ApiInstance. I've pushed a fixup for this and the other accidental s/Person/PopoloPerson/ changes that you mentioned.